regionServiceClient/etc/regionserverclnt.cfg

```diff
@@ -2,6 +2,7 @@
 api = regionInfo
 certLocation = /var/lib/regionService/certs
 regionsrv = COMMA_SEP_LIST_OF_CLOUD_SPECIFIC_REGION_SERVER
+use_metadata_server = OPTIONAL_URL_FOR_METADATA_SERVER_SMT_INFO
```

Comment:
`use_metadata_server` sounds like it takes a boolean argument. Consider just `metadata_server` instead?